tutorials/intro/40_sensor_locations.py

```diff
@@ -98,54 +93,51 @@
 # Controlling channel projection (MNE vs EEGLAB)
 # ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# Channel positions in 2d space are obtained by projecting their actual 3d
-# positions using a sphere as a reference. Because ``'standard_1020'`` montage
-# contains realistic, not spherical, channel positions, we will use a different
-# montage to demonstrate controlling how channels are projected to 2d space.
+# Channel positions in 2D space are obtained by projecting their actual 3D
+# positions on a sphere. Because the ``'standard_1020'`` montage
```

Comment:
```suggestion
# Channel positions in 2D space are obtained by projecting their actual 3D
# positions onto a sphere, then projecting the sphere onto a plane.
# Because the ``'standard_1020'`` montage
```